keyrod.gemspec

```diff
@@ -0,0 +1,24 @@
+Gem::Specification.new do |spec|
+  spec.name          = 'keyrod'
+  spec.version       = '0.0.0'
+  spec.summary       = 'Keyrod CLI'
+  spec.description   = 'CLI for authenticating OIDC clients in EGI Federated Cloud'
+  spec.authors       = ['Cuong Duong Tuan']
+  spec.email         = 'cduongt@cesnet.cz'
+  spec.files         = Dir['lib/**/*.rb', 'config/*.yml']
```

Comment:
I'm affraid this will lead to omitting important files from the gem in the future. We usually list files recognized by Git to populate this field. But it might be an overkill here. So it's up to you, either change it or keep an eye on in it in the future.